test/api/requests/build.test.js

```diff
@@ -508,7 +508,7 @@ describe('Build API', () => {
         .post(`/v0/build/${options.build.id}/status/${buildToken}`)
         .type('json')
         .send({
-          status: options.status,
+          status: encode64(options.status),
```

Comment:
Same as above, I don't think the status is base64 encoded

Reply:
encoding pushed in garden build